submission/persistence/src/main/kotlin/ac/uk/ebi/biostd/persistence/service/SubmissionRepository.kt

```diff
@@ -11,8 +11,8 @@ class SubmissionRepository(
     fun getByAccNo(accNo: String) =
         submissionDbMapper.toSubmission(submissionRepository.getByAccNoAndVersionGreaterThan(accNo))
 
-    fun getExtendedByAccNo(accNo: String) =
-        submissionDbMapper.toExtSubmission(submissionRepository.getByAccNoAndVersionGreaterThan(accNo))
+    fun getExtendedByAccNo(accNo: String, loadRefFiles: Boolean = false) =
+        submissionDbMapper.toExtSubmission(submissionRepository.getByAccNoAndVersionGreaterThan(accNo), loadRefFiles)
```

Comment:
I think we need to change the current approach of loading files in mapper by using hibernate lazy loading as is not clear in code (as is implicit) that the system is actually performing an extra query to load data.  My suggestion is to made this part of the code explicit as latter we will also add some custom logic here (to be hable to handle huge files) so I think we should propagate to mapper a map of library files that we load here in the repository.